incubator/gogs/Chart.yaml

```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 description: 'Gogs: Go Git Service'
 name: gogs
-version: 0.7.4
-appVersion: 0.11.66
+
```

Comment:
this empty line is still here?

Reply:
I dont know why, I recreate a new PR with a clean fork